helm-charts/example-apiserver/templates/konk.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: konk.infoblox.com/v1alpha1
+kind: Konk
+metadata:
+  name: {{ .Release.Name }}-konk
```

Comment:
I don't think we should include the "-konk" suffix whenever we create a konk. Its difficult to predict what the behavior would be if someone creates a konk without the "-konk" suffix.

Reply:
We can't have duplicate release names on a cluster, so if they create a Konk and a KonkService with exactly the same name there will be a conflict. This is the benefit of including `-konk` in the name. I don't have a better solution.

If we want to enforce this, we could make it part of the CRD validation.

Reply:
the CR name becomes the helm release name. We need the suffix to ensure Konk release do not conflict with Konkservice releases